src/utils/levenshtein-search.ts

```diff
@@ -46,8 +54,33 @@ export function* fuzzySearch(needle: string, haystack: string, maxDist: number)
   } else if (ngramLen >= 10) {
     yield* fuzzySearchNgrams(needle, haystack, maxDist);
   } else {
-    yield* fuzzySearchCandidates(needle, haystack, maxDist);
+    const generator = fuzzySearchCandidates(needle, haystack, maxDist);
+
+    if (debugFlag) {
+      for (const match of generator) {
+        console.log('search match', match);
```

Comment:
Extra console log?

Reply:
This was already there, I just moved it from fuzzySearchCandidates.

Reply:
Oh I am a dumb dumb. I see the condition now.